packages/core-utils/src/provider.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Provider Utilities
+ */
+
+import { ethers } from 'ethers'
+import { Provider } from '@ethersproject/providers'
+
+// Copied from @ethersproject/providers since it is not
+// currently exported
```

Comment:
PR to export the interface here: https://github.com/ethers-io/ethers.js/pull/2121